fiware-region-sanity-tests/commons/results_analyzer.py

```diff
@@ -73,18 +73,20 @@ def get_results(self):
             status = TEST_STATUS_OK
             child_node_list = testcase.childNodes
             if child_node_list is not None and len(child_node_list) != 0:
-                if child_node_list[0].localName in [CHILD_NODE_FAILURE, CHILD_NODE_ERROR]:
+                if child_node_list[0].localName in [CHILD_NODE_FAILURE, CHILD_NODE_ERROR, None]:
                     status = TEST_STATUS_NOT_OK
                 elif child_node_list[0].localName == CHILD_NODE_SKIP:
```

Comment:
The "legal" possible values are the 3 CHILD_NODE_* constants; ``None`` has been considered to cover other unexpected scenarios.